.github/workflows/e2e-parallel-full.yml

```diff
@@ -0,0 +1,172 @@
+
+jobs:
+  deploy:
+    name: Run e2e test
+    runs-on: ubuntu-latest
+
+    # These permissions are needed to interact with GitHub's OIDC Token endpoint.
+    permissions:
+      id-token: write
+      contents: read
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - example_path: examples/analytics/emr-on-eks
+            tenant_name: "emr-eks"
+          - example_path: examples/analytics/spark-k8s-operator
+            tenant_name: "spark"
+          # This is example that we can use test from deploy folder, see logic in the pre setup step
+          # - example_path: deploy/e2e/vpc
+          #   tenant_name: "vpc"
+          - example_path: examples/multi-tenancy-with-teams
+            tenant_name: "teams"
+          - example_path: examples/gitops/argocd
+            tenant_name: "argo"
+          - example_path: examples/crossplane
+            tenant_name: "crossplane"
+          - example_path: examples/aws-efs-csi-driver
+            tenant_name: "efs"
+          - example_path: examples/eks-cluster-with-new-vpc
+            tenant_name: "new-vpc"
+          - example_path: examples/fully-private-eks-cluster
+            tenant_name: "private"
+            deployment_order:
+              [
+                "module.e2e-test.module.aws_vpc",
+                "module.e2e-test.module.vpc_endpoint_gateway",
+                "module.e2e-test.module.vpc_endpoints",
+                "module.e2e-test.module.aws-eks-accelerator-for-terraform",
+              ]
+          - example_path: examples/game-tech/agones-game-controller
+            tenant_name: "agones"
+          - example_path: examples/ingress-controllers/nginx
+            tenant_name: "nginx"
+          - example_path: examples/karpenter
+            tenant_name: "karpenter"
+          - example_path: examples/node-groups/managed-node-groups
+            tenant_name: "managed"
+          - example_path: examples/node-groups/self-managed-node-groups
+            tenant_name: "self"
+          - example_path: examples/node-groups/windows-node-groups
+            tenant_name: "windows"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Pre Setup
+        id: pre-setup
+        run: |
+          mkdir -p deploy/e2e/gh-e2e-test
+          if [[ ${{ matrix.example_path }} == deploy/* ]]
+          then
+            echo "Skipping pre-setup for ${{ matrix.example_path }}"
+            cp -R ${{ matrix.example_path }}/* deploy/e2e/gh-e2e-test/
+          else 
+            echo "Running pre-setup for ${{ matrix.example_path }}"
+            mkdir -p deploy/e2e/gh-e2e-test
+            cp -R deploy/e2e/gh-e2e-template/* deploy/e2e/gh-e2e-test/
+            sed -i "s!REPLACE_ME!${{ matrix.tenant_name }}!g" deploy/e2e/gh-e2e-test/base.tfvars
+            sed -i "s!TF_STATE_PATH!${{ matrix.example_path }}!g" deploy/e2e/gh-e2e-test/backend.conf
```

Comment:
`backend.conf` file is missing in the PR. Is this file meanto be part of `deploy/e2e/gh-e2e-template/`?

Reply:
Good catch! fixing.